docs/pages/web/link.js

```diff
@@ -278,6 +277,44 @@ Please note: Disclaimers and links taking users to Pinterest subsites don't need
           <MainSection.Card
             type="do"
             description={`
+Add clarity to external links through explicit link texts and predictable destinations instead of overusing external icons when the link is surrounded by text content. If not possible, limit it to a maximum of two external icons per paragraph.
```

Comment:
grammar nits:
- "…through explicit link _text_ and predictable…"
- "If not possible, _limit to_ a maximum of…"